### ISSUE TITLE
Format "Gen 9 Metronome" missing in @pkmn/sim npm package

### ISSUE THREAD

**Issue by Pauyy** [smogon/pokemon-showdown#10198]
tl;dr: using `@pkmn/sim` it is not possible to simulate Gen 9 Metronome Battles because the format does not exist in this package. Is this intended? If it is intended, what must I do to simulate Gen 9 Metronome Battles?

I wanted to simulate some Gen 9 Metronome Battles. Using the `pokemon-showdown` npm package everything worked. 
But that package is outdated so I switched to `@pkmn/sim`. After updating my code the simulation worked in a way.
A battle could be generated, but not a Gen 9 Metronome Battle. 
Looking into `node_modules\@pkmn\sim\build\cjs\config\formats.js` or `node_modules\@pkmn\sim\build\esm\config\formats.js` a lot of Formats are present but not the Gen 9 Metronome Format.
Is it intended that the `@pkmn/sim` package does not support Gen 9 Metronome Battles?
If it is intended, what must I do to simulate Gen 9 Metronome Battles?

Manually modifying the `node_modules\@pkmn\sim\build\esm\config\format.js` works, as well as using `"pokemon-showdown": "git://github.com/smogon/pokemon-showdown.git#master"` as a dependency, but these workarounds do not seem elegant.

**Comment by mia-pi-git** [comment]
Hi - this is wrong repository for your issue (you want https://github.com/pkmn/ps/issues/28, which scheibo has already made for precisely this.)

**Comment by Pauyy** [comment]
thanks for the clarification
and thanks for the new issue
